### ISSUE TITLE
Timeline media - accessibility issues

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1330]
This page is really inaccessible for keyboard only and screen reader users.

- Tabbing often takes you to offscreen elements.
- You can't use the next / previous buttons shown on screen or the items on the timeline to change to a different piece of info (they are just divs and don't have a tab index)
- When tabbing through content you sometimes end up in a weird in-between state where slides are half off the screen
- Images often don't appear and just show as 'loading' - they show ok when using mouse to navigate to a slide

**Comment by FayCross** [comment]
For info, these issues are noted in the 3.14 accessibility statement:

The tab and/or arrow keys can be used to navigate through events on the XOT project’s Timeline Media page. However, when the focus reaches the content related to each timeline event, the focus is frequently offscreen and not visible to the end-user. This fails [2.4.7 (Focus Visible)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=144%2C1410%2C211%2C222%2C247#focus-visible). 

For a better user-experience, use the arrow keys to move through the timeline events and tab / shift-tab to move through the content related to that event.
